.eslintignore

```diff
@@ -1,3 +1,4 @@
 bin
 package-lock.json
-node_modules
\ No newline at end of file
+node_modules
+webpack.config.js
```

Comment:
Why can't the webpack file be linted?

Reply:
Updated so it's linting it again - had to add a couple lint overrides but should be good 👍